.travis.yml

```diff
@@ -1,9 +1,9 @@
 sudo:     required
 language: go
-go:       "1.11"
+go:       "1.14"
 services: redis-server
 before_install: sudo apt-get update -y && sudo apt-get install stunnel4 -y
-install: make bootstrap bootstrap_redis_tls
+install: make bootstrap_redis_tls
```

Comment:
why is bootstrap gone here, but still present in the Makefile? Should mockgen install remain part of bootstrap tests?

Reply:
Thanks for catching, at one point I removed bootstrap entirely, then put it back. I'm going to keep it there so I will restore to the travis file.